src/main/drivers/vtx_common.h

```diff
@@ -28,21 +28,25 @@ typedef enum {
 
 struct vtxVTable_s;
 
+typedef struct vtxDeviceCapability_s {
+    uint8_t bandCount;
+    uint8_t channelCount;
+    uint8_t powerCount;
+} vtxDeviceCapability_t;
+
 typedef struct vtxDevice_s {
     const struct vtxVTable_s *vTable;
 
-    uint8_t numBand;
-    uint8_t numChan;
-    uint8_t numPower;
+    vtxDeviceCapability_t capability;
 
-    uint16_t *freqTable;  // Array of [numBand][numChan]
-    char **bandNames;    // char *bandNames[numBand]
```

Comment:
I think these strings belong to each device, thus the driver level is the correct place. This will be more apparent when we add power levels and their names for RTC6705; each different implementation of VTX has different RF power and associated names and they will be declared/defined in the driver level. All the rest of the system, most notably IO and CMS, will be free from semantics of RF power indices.